x-pack/plugins/ml/public/application/routing/routes/data_frame_analytics/analytics_job_exploration.tsx

```diff
@@ -50,26 +49,8 @@ const PageWrapper: FC<PageProps> = ({ location, deps }) => {
   );
 
   const [globalState] = useUrlState('_g');
-
-  const locator = useMlLocator();
-  const {
-    services: {
-      application: { navigateToUrl },
-    },
-  } = useMlKibana();
-
-  const redirectToAnalyticsManagementPage = async () => {
-    if (!locator) return;
-    const url = await locator.getUrl({ page: ML_PAGES.DATA_FRAME_ANALYTICS_JOBS_MANAGE });
-    await navigateToUrl(url);
-  };
-
-  const jobId: string = globalState.ml.jobId;
-  const analysisType: DataFrameAnalysisConfigType = globalState.ml.analysisType;
-
-  if (!analysisType) {
-    redirectToAnalyticsManagementPage();
-  }
+  const jobId: string = globalState?.ml.jobId;
+  const analysisType: DataFrameAnalysisConfigType = globalState?.ml.analysisType;
```

Comment:
This can be removed because now if we don't have a defined job id and analysis type, the user selects a job from the job selector flyout and those values get populated with from job is chosen.